src/js/index.js

```diff
@@ -49,7 +108,6 @@ function handleTags() {
 	const tags = selectr.getValue();
```

Comment:
**EDIT: won't work as the list is joined.**

Here also, let's use the getter you wrote.

```suggestion
	const tags = parameters.tags.getValue();
```